config/verificationMail.js

```diff
@@ -26,9 +27,9 @@ const config = {
     const source = fs.readFileSync(path.join(__dirname, '../template/activation.hjs'), 'utf8');
     const template = Handlebars.compile(source);
 
-    const { name, id, email } = params;
-
-    const url = `http://localhost:3000/api/v1/activation/${id}`;
+    const { name, email } = params;
+    const tokenToSend = await jwt.sign({ email }, process.env.SECRET);
+    const url = `https://ah-kg-avengers-backend-staging.herokuapp.com/api/v1/activation/${tokenToSend}`;
```

Comment:
@niyobobo What if you use an environment variable to load this URL? Here you are fixing it to Heroku, then if I need to verify a user on the localhost I will be forced to do it manually in the database. I suppose that you check the **NODE_ENV** to decide what the URL variable will be.

Reply:
@avpaul Thank you for the feedback and good idea. I changed this.